### DISCUSSION TITLE
Why is there no getRSSI for SX1276 chipset?

### DISCUSSION THREAD

**Discussion by MrSniffer** [jgromes/RadioLib#577] [Q&A] [answered]
AFAIK there is no getRSSI() method for the SX1276 (TTGO board variant). Mistry implements packetRSSI() but I don't see a comparable method in RadioLib. It appears to me from the Semtech Datasheet that the chip does supply RSSI in a register.

**Comment by jgromes** [comment, marked as answer]
There's the `getRSSI()` method, which returns the RSSI of the last received packet if that's what you mean. The examples (e.g. SX127x_Recieve) show its usage ;)

**Reply by MrSniffer** [reply]
It seems to be omitted from the documentation, e.g. here:
https://jgromes.github.io/RadioLib/class_s_x127x.html

**Reply by jgromes** [reply]
Documentation is generated automatically - it's just in a different class: https://jgromes.github.io/RadioLib/class_s_x1278.html

There's a difference in RSSI calulcation between SX1272/73 and SX1276/77/78/79, that's why it's in the SX1272/SX1278 classes (from which other module classes derive).
